pkg/controller/keyvault/ingress_secret_provider_class_test.go

```diff
@@ -316,47 +317,16 @@ func TestIngressSecretProviderClassReconcilerInvalidURL(t *testing.T) {
 }
 
 func TestIngressSecretProviderClassReconcilerBuildSPCInvalidURLs(t *testing.T) {
-	i := &IngressSecretProviderClassReconciler{
-		ingressManager: NewIngressManagerFromFn(func(ing *netv1.Ingress) (bool, error) {
-			if *ing.Spec.IngressClassName == spcTestIngressClassName {
-				return true, nil
-			}
-
-			return false, nil
-		}),
-	}
-
 	invalidURLIng := &netv1.Ingress{
 		Spec: netv1.IngressSpec{
 			IngressClassName: &spcTestIngressClassName,
 		},
 	}
 
-	t.Run("missing ingress class", func(t *testing.T) {
```

Comment:
why are these test cases removed?

Reply:
The logic to check the ingress class name was for the isManaging check. Since I moved that logic to the placeholderPodCleanCheck I moved these tests there